components/data/librarysize/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { MetaNode } from '@/spec/metanode'
+import python from '@/utils/python'
+import { GeneCountMatrix } from '@/components/data/gene_count_matrix'
+import { norm_icon } from '@/icons'
+import { PlotlyPlot } from '@/components/viz/plotly'
+
+
+
+export const librarysize = MetaNode('librarysize')
+  .meta({
+    label: 'librarysize',
+    description: 'librarysize',
+    icon: [norm_icon],
+  })
+  .inputs({ matrix: GeneCountMatrix })
+  .output(PlotlyPlot)
+  .resolve(async (props) => await python(
+    'components.data.librarysize.createlibrarysize',
+    { kargs: [props.inputs.matrix]  },
+  ))
+  .story(props =>
+    `librarysize.`
+  )
```

Comment:
Please make more useful label, descriptions, & stories. See how they look on the main UI.
- label is the card title, and lets the user know, at a glance, what it does
- description elaborates slightly on what it does
- story describes what was done if it was clicked as a sentence in a methods section of a paper (including citations where applicable)